infrastructure/database.tf

```diff
@@ -19,6 +19,7 @@ module "postgresql" {
   ]
   pgsql_version        = "11"
   product              = var.product
+  backup_retention_days= "${var.database_backup_retention_days}"
```

Comment:
To be consistent, can you put a single space before the equals sign and then line up the admin_user_object_id, business_area, common_tags, component, env, pgsql_version and product equals signs with it?